docs/content/how-to-guides/configure-sources/configure-dataverse-source/_index.md

```diff
@@ -3,99 +3,168 @@ type: "docs"
 
-1. If you have access to multiple tenants, use the Settings icon  in the top menu to switch to the tenant in which you want to register the application from the Directories + subscriptions menu.
+## Source Definitions
+The YAML file passed to `drasi apply` can contain one or more Source definitions.
 
-1. Browse to Identity > Applications > App registrations and select New registration.
+Here is an example of a Dataverse source definition:
 
-1. Enter a display Name for your application. Users of your application might see the display name when they use the app, for example during sign-in. You can change the display name at any time and multiple app registrations can share the same name. The app registration's automatically generated Application (client) ID, not its display name, uniquely identifies your app within the identity platform.
+```yaml
+kind: Source
+apiVersion: v1
+name: my-source
+spec:
+  kind: Dataverse
+  identity:
+    kind: MicrosoftEntraWorkloadID
+    clientId: <client-id>
+  properties:
+    endpoint: https://<your-org>.api.crm.dynamics.com
+    entities: <logical_name_of_table1>,<logical_name_of_table2>
+    maxInterval: <max-interval-in-seconds>
+```
 
-1. Select Register to complete the initial app registration.
 
-When registration finishes, the Microsoft Entra admin center displays the app registration's Overview pane. You see the Application (client) ID. Also called the client ID, this value uniquely identifies your application in the Microsoft identity platform.
+In the Source resource definition:
 
-Next, you need to add credentials to the application. Credentials allow your application to authenticate as itself, requiring no interaction from a user at runtime.
+- **apiVersion** must be **v1**
+- **kind** must be **Source**
+- **name** is the **id** of the Source and must be unique. This id is used in a Continuous Query definitions to identify which Sources the Continuous Query subscribes to for change events.
+- **spec.kind** must be **Dataverse**
 
-1. In the Microsoft Entra admin center, in App registrations, select your application.
-1. Select Certificates & secrets > Client secrets > New client secret.
-1. Add a description for your client secret.
-1. Select an expiration for the secret or specify a custom lifetime.
-1. Select Add.
-1. Record the secret's value for use in your client application code. This secret value is never displayed again after you leave this page.
 
-For more information see [Use OAuth authentication with Microsoft Dataverse](https://learn.microsoft.com/en-ca/power-apps/developer/data-platform/authenticate-oauth)
+The following table describes the Dataverse specific properties:
+|Property|Description|
```

Comment:
Did we remove the secret based auth completely?

Reply:
We actually still have support for using secret based auth but I wasn't able to verify it due to being blocked by conditional access policy. Will add this part back if I managed to get it working

Reply:
@danielgerlag  Added back the docs on using sercret based auth